yamux/src/connection.rs

```diff
@@ -103,13 +103,16 @@ use closing::Closing;
 use futures::stream::SelectAll;
 use futures::{channel::mpsc, future::Either, prelude::*, sink::SinkExt, stream::Fuse};
 use nohash_hasher::IntMap;
+use parking_lot::Mutex;
 use std::collections::VecDeque;
 use std::task::{Context, Waker};
 use std::{fmt, sync::Arc, task::Poll};
 
 use crate::tagged_stream::TaggedStream;
 pub use stream::{Packet, State, Stream};
 
+use self::stream::Shared;
```

Comment:
Can we express this as a `crate::` import and merge it with the others above?

Reply:
Actually the import is no longer needed after using qualified `stream::Shared`, removed